services/galley/src/Galley/API/Query.hs

```diff
@@ -541,22 +544,37 @@ getConversationByReusableCode lusr key value = do
         }
 
 -- FUTUREWORK(leif): refactor and make it consistent for all team features
-ensureGuestLinksEnabled ::
+ensureGuestLinksEnabledWithError ::
   forall r.
   ( Member (Error ConversationError) r,
+    Member (Error CodeError) r,
     Member TeamFeatureStore r,
     Member (Input Opts) r
   ) =>
-  Data.Conversation ->
+  Either ConversationError CodeError ->
```

Comment:
```suggestion
  Either ConversationError CodeError {- FUTUREWORK: "a `Member` [ConversationError, CodeError] => e"? -} ->
```

this may work or not, and i don't think we should spend the time to sort this out now, but if(f) you make any more commits you might as well leave this comment?